RetailSuiteAccounting/Operation/02_ReviewAccount.md

```diff
@@ -0,0 +1,75 @@
+[!!User Interface balances](../UserInterface/01b_Balances.md)  
+[!!User Interface account sheet](../UserInterface/01c_AccountSheet.md)  
+[!!User Interface accounts](../UserInterface/02b_Accounts.md)  
+[!!Manage the accounts](../Integration/03_ManageAccounts.md)  
+
+
+# Review an account
+
+The *Book* menu entry in the *Accounting* module contains specific functions to display different views of the data available in the system. All bookings recorded in the *Accounting* module can be filtered per account and period and visually displayed for better clarity or for checking purposes.
+
+
+## Access the balance view of an account
+
+ The *BALANCES* tab offers a T-account view of the current credit and debit situation, and the resulting balance of a selected account.
+
+ The balance value represents the difference between credits and debits in the account. If the debit and credit totals are equal, the balance is settled. If a difference between them exists, there is an outstanding balance to be settled, for example, a payment must be made to bring the account balance to zero.
+
+ > [Info] When clicking the *BALANCES* tab, a blank tab and the message "No account selected" may be displayed. An account must be selected to display the corresponding details. If a booking is selected in the *BOOKINGS* tab, the account and contra account details corresponding to the selected booking will be displayed in the *BALANCES* tab.
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
+- A valid fiscal year is created, see [Create a fiscal year](../Integration/04_ManageFiscalYear.md#create-a-fiscal-year).
+- A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).
+
+#### Procedure
+
+*Accounting > Select fiscal year > Book > Tab BALANCES*
+
+![Balances - no account selected](../../Assets/Screenshots/RetailSuiteAccounting/Book/Balances/BalancesNoAccount.png "[Balances - no account selected]")
+
+> [Info] When clicking the *BALANCES* tab, a blank tab and the message "No account selected" may be displayed. An account must be selected to display the corresponding details.
+
+1. Enter an account number in the *Contra account* field.  
+The balance for the selected account is displayed on the left side of the workspace.
+
+2. Enter an account number in the *Account* field.  
+The balance for the selected account is displayed on the right side of the workspace.
```

Comment:
Muss ich Konto und Gegenkonto aufrufen? Oder ich welchem Fall mache ich das? -> vielleicht noch in den Einleitungssatz rein.
Ansonsten zusätzlich ein "if desired" an die satzanfänge, um zu zeigen, dass nicht Konot und gegenkonto angegeben sein müssen...